feat(curriculum): add cat photo app workshop (#55552)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698510677b5e655ba8c1017.md

---
id: 6698510677b5e655ba8c1017
title: Step 40
challengeType: 0
dashedName: step-40
---

# --description--

You can set browser behavior by adding `meta` elements in the `head`. Here's an example:

```html
<meta attribute="value">
```

> Inside the `head` element, nest a `meta` element with an attribute named `charset`. Set to the value to `utf-8` which tells the browser how to encode characters for the page. 

Reviewer: ```suggestion
Inside the `head` element, nest a `meta` element with a `charset` attribute set to the value of `utf-8`. This tells the browser how to encode characters for the page. 
```
